src/test/FuzzerImpl.cpp

```diff
@@ -745,11 +765,22 @@ TransactionFuzzer::initialize()
                     trustOp.sourceAccount.activate() = toMuxedAccount(account);
                     ops.emplace_back(trustOp);
 
+                    PublicKey issuer;
+                    FuzzUtils::setShortKey(issuer, j);
+
+                    if ((accountParameters[j].mAccountFlags &
```

Comment:
Probably want some to be authorized and some not to be authorized, right? What about authorized to maintain liabilities?

Reply:
see my other comment, balances, limits and offers are also symmetrical. I am not sure a PR is the place to brainstorm on how we should mix things up.

Reply:
> see my other comment, balances, limits and offers are also symmetrical. I am not sure a PR is the place to brainstorm on how we should mix things up.

Moved to a separate PR.  We can discuss whatever further variations we want elsewhere (issue #1376?).